deployments/scripts/deploy-rbac-staging.sh

```
@@ -0,0 +1,239 @@
+
+    kubectl wait --for=condition=available --timeout=300s deployment/rbac-token-service -n "$NAMESPACE"
+
+    log "RBAC token service is ready"
+}
+
+# Run health checks
+run_health_checks() {
+    log "Running health checks..."
+
+    # Get the service endpoint
+    SERVICE_IP=$(kubectl get service rbac-token-service -n "$NAMESPACE" -o jsonpath='{.spec.clusterIP}')
```

Shellcheck (0.10.0)

[warning] SERVICE_IP appears unused. Verify use (or export if used externally).

(SC2034)